Cargo.toml

```diff
@@ -175,7 +175,7 @@ wasm-bindgen-test = { version = "0.3.37", default-features = false }
 libc = { version = "0.2.148", default-features = false }
 
 [build-dependencies]
-cc = { version = "1.0.83", default-features = false }
+cc = { version = "1.0.94", default-features = false }
```

Comment:
What is the oldest cc-rs that is known to work for this? Because of issues like https://github.com/rust-lang/cc-rs/issues/984 I'm weary of forcing users to update to too-new version of cc-rs right now, if we can avoid it.

Reply:
VisionOS support was added in [1.0.93](https://github.com/rust-lang/cc-rs/releases/tag/1.0.93), but [1.0.94](https://github.com/rust-lang/cc-rs/releases/tag/1.0.94) includes a fix to how it integrates with `xcrun` that may be necessary. I don't see anything in the changelog after 1.0.94 related to visionos.

Reply:
Thanks. I think we need to go through the release notes of cc-rs 1.0.94 and higher, and recent issues in that crate, to verify that requiring such a new release is not going to be problematic, given recent instability regarding Apple targets in that crate.

Reply:
so what would be the best course of action here? seems like https://github.com/rust-lang/cc-rs/issues/984 is still a blocker.

original PR was merged on April 12, and release followed on April 13
https://github.com/rust-lang/cc-rs/releases/tag/1.0.93

Reply:
Maybe we could leave the cc-rs dependency as-is but add a comment above it saying cc-rs 1.0.94 or later is required for visionos support, but we're waiting to increase our dependency until https://github.com/rust-lang/cc-rs/issues/984 and maybe other issues are addressed, and also add a note to that same effect to BUILDING.md. Then we can merge this.

Reply:
yeah, a conditional dependency in `Cargo.toml` sounds like a feasible option.

@QuentinPerez wanna do it?